test/controllers/groups_controller_test.rb

```diff
@@ -5,14 +5,20 @@ class GroupsControllerTest < ActionController::TestCase
     login_as_user(:sanyi)
   end
 
-  test "show groups list" do
+  test "show all groups list" do
     result_per_page = 20
-    get :index, per: result_per_page
+    get :all, per: result_per_page
 
     assert_equal result_per_page, assigns(:groups).size
     assert_equal groups(:babhamozo).id, assigns(:groups).first.id
   end
 
+  test "show active groups list" do
+    get :index, per: 20
+
+    assert_equal 0, assigns(:groups).size
```

Comment:
20-at kerunk de 0-t kapunk? ha bovul a fixtures akkor valszeg mar nem 0-t kapunk.

Reply:
Inactive? checks the last two semesters' evaulation and now none of the test groups have any. So in my opinion, the test works fine :D

Reply:
But if sb add evaulation 'cause wanna test is, then the test will fail.